prisma/schema.prisma

```diff
@@ -0,0 +1,137 @@
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["referentialIntegrity"]
+}
+
+datasource db {
+  provider             = "mysql"
+  url                  = env("DATABASE_URL")
+  referentialIntegrity = "prisma"
+}
+
+enum GameType {
+  standard
+  rapidplay
+  blitz
+}
+
+enum Federation {
+  FIDE
+  ECF
+  USCF
+}
+
+model PlayerRating {
+  id         Int        @id @default(autoincrement())
+  createdAt  DateTime   @default(now())
+  updatedAt  DateTime   @updatedAt
+  type       GameType
+  federation Federation
+  rating     Int?
+  playerId   Int?
+  player     Player?    @relation(fields: [playerId], references: [id])
+
+  @@index([playerId])
+}
+
+model Player {
+  id              Int            @id @default(autoincrement())
+  createdAt       DateTime       @default(now())
+  updatedAt       DateTime       @updatedAt
+  firstName       String
+  lastName        String
+  middleName      String?
+  teamId          Int?
+  team            Team?          @relation(fields: [teamId], references: [id])
+  ratings         PlayerRating[]
+  fixturesAsWhite Fixture[]      @relation("PlayerWhite")
```

Comment:
This is interesting! Hadn't considered it in this way 👍🏼

Reply:
yeah we need a way to differentiate players in a fixture, seemed like a natural way to do it 🤔

Reply:
Took a similiar approach here: https://github.com/chess-centre/platform/blob/master/chess-centre-app/src/pages/App/Games.tsx#L217

This is because a game / result record is a single entry which is queried by two players via different indexes (asWhite, asBlack). 

The alternative would be to store a result as two transactions, but this seems ugly.